infoblox_client/objects.py

```diff
@@ -779,12 +779,12 @@ def ip(self, ip):
 
 class FixedAddressV4(FixedAddress):
     _infoblox_type = 'fixedaddress'
-    _fields = ['ipv4addr', 'mac', 'network_view', 'extattrs', 'network',
-               'options', 'comment']
+    _fields = ['ipv4addr', 'mac', 'network_view', 'extattrs', 'name',
+               'network', 'options', 'template', 'comment']
     _search_for_update_fields = ['ipv4addr', 'mac', 'network_view', 'network']
     _all_searchable_fields = _search_for_update_fields
     _shadow_fields = ['_ref', 'ip']
-    _return_fields = ['ipv4addr', 'mac', 'network_view', 'extattrs']
+    _return_fields = ['ipv4addr', 'mac', 'name', 'network_view', 'extattrs']
```

Comment:
return field doesn't need network?

Reply:
@jkraj I'll check this point. Was not required on my side, but could be interesting to have this field available.

Reply:
My 2 cents...

Typically, you search/create a FixedAddress specifying `network` as a criteria - i.e. you already know the network - which I assume is why 'network' was never included in `_return_fields`.

Additionally, I believe `_return_fields` only specifies the *default* set of return fields. If needed, you can always add `network` to the set of return fields when doing a get/search. (see my comment here: https://github.com/infobloxopen/infoblox-client/pull/187#issuecomment-369620183)